### DISCUSSION TITLE
Installing insiders

### DISCUSSION THREAD

**Discussion by spinningrachel** [squidfunk/mkdocs-material#2250] [Q&A]
Hi. I have an existing project I've been working on and customizing for a couple of months, and now I'd like to add/install insiders on top of it. 

I tried just following the directions in the docs but that didn't really work out ... how do I install insiders into an already existing project ... any help is much appreciated ...

**Comment by squidfunk** [comment]
There are several methods to install Insiders, depending on whether you use [`pip`](https://squidfunk.github.io/mkdocs-material/getting-started/#with-pip), [`docker`](https://squidfunk.github.io/mkdocs-material/getting-started/#with-docker) or [`git`](https://squidfunk.github.io/mkdocs-material/getting-started/#with-git). Please also read the [footnotes](https://squidfunk.github.io/mkdocs-material/getting-started/#fn:1) that are linked in the respective sections to learn about which scopes you need to grant to your GH token for access.

If you're still having trouble, please provide more details how your setup looks like and what you already tried, and how it failed.
